prepare-solvcon-dev.sh

```diff
@@ -45,11 +43,14 @@ conda update -q conda
 conda create -p ${SCSRC_WORKING}/venv-conda --no-default-packages -y python
 source activate ${SCSRC_WORKING}/venv-conda
 
+# prepare all packages to build SOLVCON
+${SOLVCON_DEVENV_SRC}/conda.sh
+${SOLVCON_DEVENV_SRC}/build-pybind11-in-conda.sh
+
 set +x
 
-# prepare all packages to build SOLVCON
-${SCSRC}/contrib/conda.sh
-${SCSRC}/contrib/build-pybind11-in-conda.sh
+# fetch SOLVCON source
+git clone https://github.com/solvcon/solvcon.git ${SCSRC}
```

Comment:
SOLVCON repository is used for testing the devenv that has been built. The building process of devenv does not really need SOLVCON repository now.